charts/router/templates/router-deployment.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if not .Values.global.experimental_native_ingress }}
```

Comment:
I think I may have said something to this effect once before, but I don't think this works... `.Values.global` looks like a reference to the "top level" configuration in the Workflow chart. This chart can't "look up" into there because... well... imagine the scenario where this chart is installed standalone (i.e. without the rest of Workflow)... there wouldn't even be anything to look up _to_.

I _think_ this repo probably does not require any changes to support the native ingress initiative.